wrapper/src/main/java/software/amazon/jdbc/wrapper/ConnectionWrapper.java

```diff
@@ -139,15 +146,31 @@ protected void init(
     }
   }
 
-  protected String getProtocol(final String url) {
+  protected DatabaseDialect getDialect(final String url, Properties info) {
+    String dialectClassName = DATABASE_DIALECT.getString(info);
+    if (dialectClassName != null ){
+      try {
+        return (DatabaseDialect) Class.forName(dialectClassName).getDeclaredConstructor().newInstance();
+      }catch (Exception ex) {
+
```

Comment:
Are we intentionally ignoring the exception? We may want to throw an exception here.